src/test/java/com/google/alpollo/AnalysisTest.java

```diff
@@ -55,6 +55,16 @@
   private static final String EMPTY_STRING = "";
   private static final String WIKI_LINK_GOOGLE = "https://en.wikipedia.org/wiki/Google";
 
+  private static final double NEUTRAL_MAGNITUDE = 0;
+  private static final double NON_NEUTRAL_MAGNITUDE = 10;
+  private static final double NEUTRAL_SCORE = 0.1;
+  private static final double NEGATIVE_SCORE = -2;
+  private static final double POSITIVE_SCORE = 2;
+
+  private static final String NEUTRAL_INTERPRETATION = "There isn't much going in this song.";
+  private static final String NEGATIVE_INTERPRETATION = "This song is predominantly negative.";
+  private static final String POSITIVE_INTERPRETATION = "This song is predominantly positive";
+  private static final String MIXED_INTERPRETATION = "This song has a lot of mixed feelings";
```

Comment:
Reuse the strings from AnalysisHelper directly instead of redefining them, it is fine to have them public as they are constants.